docs-vim.js

```diff
@@ -13,7 +14,15 @@ vim.switchToNormalMode = function () {
     docs.setCursorWidth("7px");
 };
 
+vim.switchToVisualMode = function () {
+    vim.currentSequence = "";
+    vim.mode = "visual";
+    vim.num = "";
+    docs.setCursorWidth("7px");
+};
+
 vim.switchToInsertMode = function () {
+    vim.currentSequence = "";
     vim.mode = "insert";
```

Comment:
Nit: Is it OK to reset `vim.num` here, and `vim.currentSequence` in `vim.switchToNormalMode`? We might just want to have a generic "setMode" which resets these and sets the desired mode. We can handle this in a different issue/PR though.